src/measure_theory/borel_space.lean

```diff
@@ -423,4 +456,26 @@ begin
       measurable_const }
 end
 
+lemma measurable_add {α : Type*} [measurable_space α] {f g : α → ennreal} :
+  measurable f → measurable g → measurable (λa, f a + g a) :=
+begin
+  refine measurable_of_measurable_nnreal_nnreal (+) _ _ _,
+  { simp only [ennreal.coe_add.symm],
+    exact measurable_coe.comp
+      (measurable_add (measurable_fst measurable_id) (measurable_snd measurable_id)) },
+  { simp, exact measurable_const},
```

Comment:
In general we don't want nonterminal `simp`s in mathlib. You can use `squeeze_simp` to see lemmas being used.

Reply:
All corrected. Thanks

Reply:
At some places listing all the lemmas feels cumbersome. Here is one example I encountered:
```lean
begin simp, use n, simp end
```
```lean
begin 
  simp only [mem_at_top_sets, mem_set_of_eq, nonempty_of_inhabited], 
  use n, 
  simp only [imp_self, forall_const] 
end
```
Do I still make the change?

Reply:
The last `simp` is terminal, so you could just `simp` :)